prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div {
+      display: flex;
+    }
```

Comment:
I recommend removing this rule set, as we want to be very targeted on exactly which divs to apply styling to.